atlasdb-workload-server/src/test/java/com/palantir/atlasdb/workload/transaction/WitnessedTransactionsBuilder.java

```diff
@@ -41,7 +41,9 @@ public WitnessedTransactionsBuilder(String table) {
     }
 
     public WitnessedTransactionBuilder startTransaction(long startTimestamp) {
-        timestampCounter.set(startTimestamp);
+        if (startTimestamp > timestampCounter.get()) {
+            timestampCounter.set(startTimestamp);
```

Comment:
not actionable: I was intending a getAndAccumulate/accumulateAndGet with `Long::max`, though I guess this is not going to be used in a multithreaded context!